### DISCUSSION TITLE
Create a better display of Vaal side area mods and Searing Exarch & Eater of Worlds map mod pop ups

### DISCUSSION THREAD

**Discussion by GrimDawn-max** [Lailloken/Exile-UI#111] [Ideas & Feedback]
You probably don't need more ideas to work with but there are two I came across while playing POE today

1. Vaal side area have their own set of mods. Like map mods they are ridiculously long and hard to read. Would it be possible to implement the map info box for Vaal side areas as well so a player can highlight and see any dangerous mods (like reflect damage or no regen) at a glance instead of reading a short story ?

2. Would it also be possible to create a display summary of the 2 choices you get when the Searing Exarch and Eater of Worlds pop ups when occur maps that is easier to read than the in-game one ? Like reading map and vaal side area mod list I don't have the time to stop and read these things, especially since the signs are surrounded by mobs and I'll die trying to stop and read and it's hard to read  on the run chased by mobs. This might be tricky since they randomly pop up  but here's hoping.

**Comment by Lailloken** [comment]
This would require proper OCR, as opposed to the simple image recognition the script uses currently. There are open OCR projects I could implement, but I generally want to avoid doing that. There's also the problem that this information (the text it would have to scan) is presented dynamically on the 'game-screen' (often on semi-transparent backgrounds), which would probably impact accuracy/speed quite a bit.

**Comment by GrimDawn-max** [comment]
Fair enough !
